app/src/main/java/com/codesoom/controllers/ProductController.java

```diff
@@ -0,0 +1,93 @@
+package com.codesoom.controllers;
+
+import com.codesoom.application.ProductService;
+import com.codesoom.domain.Product;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * 상품에 대한 HTTP 요청의 처리를 담당한다.
+ */
+
+@RestController
```

Comment:
```suggestion
/**
 * 상품에 대한 HTTP 요청의 처리를 담당한다.
 */
@RestController
```

JavaDoc과 코드 사이에 빈 줄을 두지 마세요.